tests/test_distributions.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+import math
+import numpy as np
+import random
+import scipy.stats as sts
+from pymc4 import distributions as dist
+from numpy.testing import assert_almost_equal
+
+def test_normal():
```

Comment:
Not sure we need these tests because they already happen in TFP, instead we should test that our parameterization matches what TFP provides.

Reply:
We need a test to match scipy parametrization since the implementation API is backend agnostic.  We then just change env var in Travis to test other backends

Reply:
I would test the change of parameters additionally

Reply:
@twiecki  so I was trying to add tests just like in [pymc3/test_distributions.py](https://github.com/pymc-devs/pymc3/blob/master/pymc3/tests/test_distributions.py) i.e using `Domains`, `Point()` etc. However @ferrine  suggested to look into the [hypothesis](https://hypothesis.readthedocs.io/en/latest/details.html) library for testing distributions. 

I will look into the library (it's supposed to integrate well with pytest) and make a draft PR of how the tests are supposed to look. Just adding discrete distributions for now. Thank you for your patience.